code/datums/wounds/cuts.dm

```diff
@@ -0,0 +1,259 @@
+		if(0 to 1.25)
+			bandage_condition = "nearly ruined "
+		if(1.25 to 2.75)
+			bandage_condition = "badly worn "
+		if(2.75 to 4)
+			bandage_condition = "slightly bloodied "
+		if(4 to INFINITY)
+			bandage_condition = "clean "
+	return "<B>The cuts on [victim.p_their()] [limb.name] are wrapped with [bandage_condition] [current_bandage.name]!</B>"
+
+/datum/wound/brute/cut/receive_damage(wounding_type, wounding_dmg, wound_bonus)
+	if(wounding_type == WOUND_SHARP)
+		blood_flow += 0.05 * wounding_dmg
+
+/datum/wound/brute/cut/handle_process()
+	if(victim.reagents && victim.reagents.has_reagent(/datum/reagent/medicine/coagulent))
+		blood_flow -= 0.25
+	if(current_bandage)
+		if(clot_rate > 0)
+			blood_flow -= clot_rate
+		blood_flow -= current_bandage.absorption_rate
+		current_bandage.absorption_capacity -= current_bandage.absorption_rate
+		if(current_bandage.absorption_capacity < 0)
+			victim.visible_message("<span class='danger'>Blood soaks through \the [current_bandage] on [victim]'s [limb.name].</span>", "<span class='warning'>Blood soaks through \the [current_bandage] on your [limb.name].</span>", vision_distance=COMBAT_MESSAGE_RANGE)
+			QDEL_NULL(current_bandage)
+			treat_priority = TRUE
+	else
+		blood_flow -= clot_rate
+
+	if(blood_flow > highest_flow)
+		highest_flow = blood_flow
+
+	if(blood_flow < minimum_flow)
+		if(demotes_to)
+			replace_wound(demotes_to)
+		else
+			to_chat(victim, "<span class='green'>The cut on your [limb.name] has stopped bleeding!</span>")
+			qdel(src)
+
+/* BEWARE, THE BELOW NONSENSE IS MADNESS. bones.dm looks more like what I have in mind and is sufficiently clean, don't pay attention to this messiness */
+
+/datum/wound/brute/cut/treat(obj/item/I, mob/user)
+	if(istype(I, /obj/item/gun/energy/laser))
+		las_cauterize(I, user)
+	else if(I.tool_behaviour == TOOL_CAUTERY || I.get_temperature() > 300)
+		tool_cauterize(I, user)
+	else if(istype(I, /obj/item/stack/medical/gauze))
+		bandage(I, user)
+	else if(istype(I, /obj/item/stack/medical/suture))
+		suture(I, user)
+
+/datum/wound/brute/cut/proc/las_cauterize(obj/item/gun/energy/laser/lasgun, mob/user)
+	var/self_penalty_mult = (user == victim ? 1.5 : 1)
+	user.visible_message("<span class='warning'>[user] begins aiming [lasgun] directly at [victim]'s [limb.name]...</span>", "<span class='userdanger'>You begin aiming [lasgun] directly at [user == victim ? "your" : "[victim]'s"] [limb.name]...</span>")
+	var/time_mod = user.mind?.get_skill_modifier(/datum/skill/medical, SKILL_SPEED_MODIFIER) || 1
+	if(!do_after(user, base_treat_time * time_mod * self_penalty_mult, target=victim, extra_checks = CALLBACK(src, .proc/still_exists)))
+		return
+	var/damage = lasgun.chambered.BB.damage
+	lasgun.chambered.BB.wound_bonus -= 30
+	lasgun.chambered.BB.damage *= self_penalty_mult
+	if(!lasgun.process_fire(victim, victim, TRUE, null, limb.body_zone))
+		return
+	victim.emote("scream")
+	blood_flow -= damage / (10 * self_penalty_mult)
```

Comment:
you, uh, might wanna make sure that this shot can cause burn damage before you make it cauterize the patient's cut(s)